libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -322,6 +322,8 @@ export interface IDateFilterOwnProps extends IDateFilterStatePropsIntersection {
     // (undocumented)
     isEditMode?: boolean;
     // (undocumented)
+    isTimeEnabledForAbsoluteRange?: boolean;
```

Comment:
Is this `isTimeEnabledForAbsoluteRange` property everywhere + making it part of the public API really necessary?
Maybe we can recognize it by parsing dateFormat string? (because if you parse the dateFormat string, you can check if it contains time or not).

And apps (AD/Dashboard component) will decide what to send to the dateFormat string, according to backend capabilities?

Or is there some particular reason, why we cannot do it this way?

Reply:
There are several things we had to consider and led into this implementation. In AD, dateFormat represents format for calendar only. In KD, it also represents title within FilterBarButton (but it is not clear whether it will be there forever). There are also challenges related to time identification within the dateFormat since it can be anywhere and it can have its specific format. We have to define some rules to follow then.